docs/user/alerting/action-types/resilient.asciidoc

```diff
@@ -65,30 +65,11 @@ API key secret::  The authentication key secret for HTTP Basic authentication.
 
 IBM Resilient actions have the following configuration properties:
 
-Subaction::        The subaction to perform: `pushToService`, `getFields`, `incidentTypes`, or `severity`.
-Subaction params:: The parameters of the subaction.
-
-==== `pushToService` subaction configuration
-
-Incident:: The IBM resilient incident has the following properties:
-* `name` - A name for the issue, used for searching the contents of the knowledge base.
-* `description` - The details about the incident.
-* `externalId` - The ID of the incident in IBM Resilient. If present, the incident is updated. Otherwise, a new incident is created.
-* `incidentTypes` - An array with the IDs of IBM Resilient incident types.
-* `severityCode` - The IBM Resilient ID of the severity code.
-Comments:: A comment in the form of `{ commentId: string, version: string, comment: string }`.
-
-===== `getFields` subaction configuration
-
-The `getFields` subaction has not parameters. Provide an empty object `{}`.
-
-===== `incidentTypes` subaction configuration
-
-The `incidentTypes` subaction has no parameters. Provide an empty object `{}`.
-
-===== `severity` subaction configuration
-
-The `severity` subaction has no parameters. Provide an empty object `{}`.
+Incident types:: The incident types of the incident.
```

Comment:
```suggestion
Incident types:: The type of the incident.
```

Reply:
Can you give an example?

The type of the incident, for example, ....

Reply:
I'm wondering about the order.  Are they in the order they appear in the UI?

Reply:
Yes, these are in the order they appear in the UI